silo-vaults/contracts/MetaMorpho.sol

```diff
@@ -851,4 +862,39 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
     function _expectedSupplyAssets(IERC4626 _market, address _user) internal view virtual returns (uint256 assets) {
         assets = _market.convertToAssets(_market.balanceOf(_user));
     }
+
+    function _update(address from, address to, uint256 value) internal virtual override {
+        super._update(from, to, value);
+
+        if (value == 0) return;
+
+        // after token transfer/mint/burn
+
+        require(!_lock, ErrorsLib.NotificationDispatchError());
+        _lock = true;
+
+        address[] memory receivers = INCENTIVES_MODULE.getNotificationReceivers();
+
+        uint256 total = totalSupply();
+        uint256 senderBalance = from == address(0) ? 0 : balanceOf(from);
+        uint256 recipientBalance = to == address(0) ? 0 : balanceOf(to);
+
+        for(uint256 i; i < receivers.length; i++) {
+            // TODO do we need delegate call here?
```

Comment:
We don't need a delegate call here. Delegate call only to the incentives claiming logic.
```suggestion
```

Reply:
TODO removed